tests/data/minitest/record_test_result.json

```diff
@@ -7,6 +7,10 @@
         {
           "type": "testcase",
           "name": "test_should_save_user_with_name"
+        },
+        {
+          "type": "class",
+          "name": "UserCopyTest"
```

Comment:
This components will be `file=test/models/user_copy_test.rb#testcase=test_should_save_user_with_name#class=UserCopyTest` as test path

but I think `file=test/models/user_copy_test.rb#class=UserCopyTest#testcase=test_should_save_user_with_name` is correct.

Could you check it?

Reply:
Good catch. Let me fix it. Thanks!

Reply:
[7bd4ddc](https://github.com/launchableinc/cli/pull/846/commits/7bd4ddc3d85994f4920bbb649796a9210ad6318d)